src/mod_global_variables.f08

```diff
@@ -259,7 +262,7 @@ subroutine set_gamma(gamma_in)
     real(dp), intent(in)    :: gamma_in
 
     if (incompressible) then
-      gamma = 1.0d8
+      gamma = 1.0d6
```

Comment:
you'll have to edit the test `test_gamma_incompressible` in `tests/unit_tests/mod_test_variables.pf` if this value is changed